### PR TITLE
Add a recipe for Symfony mailer

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by fabpot** [comment]
The issue is that there is no way right now to have an alias that points to a different recipe depending of the version of Symfony used.

**Comment by fabpot** [comment]
It's even worse than that. After merge, `mailer` will always reference `symfony/mailer`.

**Comment by fabpot** [comment]
Ready!

File: symfony/mailer/4.3/post-install.txt [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    For instance, <info>composer require mailgun-bridge</> for Mailgun.
+
+  * If you want to send emails asynchronously:
+
+    1. Install the messenger component by running <info>composer require messenger</>;
+    2. Add <info>'Symfony\Component\Mailer\Messenger\SendEmailMessage': amqp</> to the
+       <info>config/packages/messenger.yaml</> file under <info>framework.messenger.routing</>
+       and replace <info>amqp</> with your transport name of choice.
+
+  * <fg=blue>Read</> the documentation at <comment>https://symfony.com/doc/master/mailer.html</>
```
Comment (sroze) [inline symfony/mailer/4.3/post-install.txt]: @fabpot this is not yet available though.

Comment (fabpot) [reply symfony/mailer/4.3/post-install.txt]: I know :)

**Pull request merged**